Scripts/ScaleMovement.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ScaleMovement : MonoBehaviour
+{
+    public float rotateSpeed = 5;
+    // Start is called before the first frame update
+    void Start()
+    {
+        

```

Comment:
Don't leave empty functions, has some cost but the compiler will probably delete this for you. But is a good practice to not leave use empty functions